src/app/components/menu/image.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+const Images = ({ imageSrc = null, iconList = [] }) => {
+  return (
+    <div className="images">
+       {imageSrc && <img src={imageSrc} alt="Logo"/>}
+      <div className="iconsContainer">
+        {iconList.map((icon, index) => (
+          <FontAwesomeIcon key={index} icon={icon} />
+        ))}
+      </div>
```

Comment:
Do czego to służy?

Reply:
renduje po wszystkich ikonkach z tablicy iconList i wyświetla te, ktore przekaze jako icon